sphere-models/src/main/java/io/sphere/sdk/types/Custom.java

```diff
@@ -0,0 +1,5 @@
+package io.sphere.sdk.types;
+
+public interface Custom {
```

Comment:
sorry I made a mistake in my first comment.
The json is actually:
```
{
  [...],
  "custom": {
    "type": { ... },
    "fields": {
      "fieldA": "valueA"
    }
  }
}
```

Replacing `custom` with `customizable` or `customizableFields` in the JSON is not very pretty.
We designed the JSON first. In that case, the name does not apply very well to OO-languages. But it is the best solution we found.
If you want to, you can change the class name. Just be sure the serialize/deserialize it to `custom`.